package.json

```diff
@@ -43,7 +43,8 @@
     "react-native-toast-message": "^2.1.6",
     "react-native-vector-icons": "^9.2.0",
     "react-native-webview": "^11.26.1",
-    "react-query": "^3.39.3"
+    "react-query": "^3.39.3",
+    "rn-emoji-keyboard": "^1.2.2"
```

Comment:
This package doesn't appear to be used at all.

Reply:
@namick @evanheisler whats the best way / practice that we have to handle upstream/downstream dependencies?

Reply:
Right now even if the SDK version doesn't change, a fresh `yarn install` is going to grab the latest minor/patch and want to install peer dependencies. It makes it a bit confusing in CR but not a big deal for now — a commit comment may help pointing to the SDK change. We could consider pinning the SDK package version. Then peer dependencies would be seen alongside the SDK version bump.